### ISSUE TITLE
any operators with '>' or '<' like e.g.  '==>' generate '== &amp;gt;'

### ISSUE THREAD

**Issue by MARTIMM** [Raku/doc#3790]
## Html escaping problem

The header at the top of a doc page shows e.g. **infix &amp;lt;==** for an explanation of `<==`. This should show **infix <==**.  This goes for all operators where a `<` or `>` is involved.

Example pages are [infix =>](https://docs.raku.org/routine/=%3E), [syntax <<](https://docs.raku.org/syntax/%3C%3C) and [infix <==](https://docs.raku.org/routine/%3C==).

**Comment by JJ** [comment]
Yep, we'll have to take care of that at the Documentable level. Thanks for letting us know.

**Comment by labster** [comment]
Also applies to https://docs.raku.org/routine/&& as `infix &amp;&amp;`.  Just noting it's not specifically `<>` but includes other escaped characters.

**Comment by 2colours** [comment]
@coke could you please move this to [doc-website](https://github.com/Raku/doc-website/issues)?

I don't know if the title (as in the text in the tab of the browser) was meant originally as well but it seems that the content is okay, only the title needs some extra help. It doesn't sound difficult to fix.
